guides/doc-Upgrading_and_Updating/topics/post_upgrade_template_checks.adoc

```diff
@@ -0,0 +1,37 @@
+[id="post_upgrade-updating-templates-parameters_{context}"]
+= Updating Templates, Parameters, Lookup Keys and Values
```

Comment:
This was dropped in 23859bf7303b6822de5c2e1ddc7f6cf6bb49e1ad and should stay out.